src/views/shared/typography/typography.view.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from "react";
+
+import useTypographyStyles from "src/views/shared/typography/typography.styles";
+
+interface TypographyProps {
+  type: "h1" | "h2" | "body1" | "body2";
+  className?: string;
+}
+
+const Typography: FC<TypographyProps> = ({ type, className, children }) => {
```

Comment:
I prefer Text instead Typography

Reply:
Text is very broad, Typography is how it's called in any UI library like Bootstrap or Material